server/data/seeds/005_followups.js

```diff
@@ -1,19 +1,15 @@
 const faker = require('faker');
 
-const followups = [];
+const refreshrs = [];
 
 for (let i = 0; i < 100; i++) {
-  followups.push({
+  refreshrs.push({
     date: faker.date.future(),
-    question: Math.floor(Math.random() * 100),
+    question: Math.ceil(Math.random() * 100),
```

Comment:
Just curious about the Math.random(), is it because we will have a question bank of 100 questions or something?

Reply:
Yep, exactly! We can up it, but I originally seeded the questions table with 100 questions, so the Math.random() just gets a random number and prevents foreign key constraints.